Data publishing via EBI FTP as LSF Job

mbdebian opened the pull request:
This pull requests re-implements data synchronization from Google Cloud Storage Bucket into EBI file system and FTP service as an LSF job.
It solves opentargets/issues#2581

JarrodBaker reviewed (changes requested):
Can you please: 

- delete the commented out code on line 94, and the (I think) old 'todo' on line 103.
- Add something to the readme about how to run this. I'm really unfamiliar with the EBI infrastructure / LSF. Do I just have to log into a node and execute the make file? 

Thanks!

mbdebian commented:
I have updated the pull request, except for the 'TODO', which is a placeholder for the filesystem synchronization command that I'm waiting on IT Support for.

The current code does everything to publish the data in the FTP, but there is a special command in the LSF to trigger the internal file system synchronization, so changes are visible via the FTP service quicker.

As per the README file, the mechanism for running the FTP data publishing is the same, but I have added a note to show that it has to be run from within an EBI login node in the EBI infrastructure.

Cheers!

> Can you please:
> 
> * delete the commented out code on line 94, and the (I think) old 'todo' on line 103.
> * Add something to the readme about how to run this. I'm really unfamiliar with the EBI infrastructure / LSF. Do I just have to log into a node and execute the make file?
> 
> Thanks!
